Cannot build  individual LaTeX documents

When trying to build [individual LaTeX documents](https://github.com/input-output-hk/cardano-ledger#building-individual-latex-documents) I get the following errors:

```
#> nix-shell --pure --run make
trace: Not using IOHK default nixpkgs (use 'niv drop nixpkgs' to use default for better sharing)
trace: haskell-nix.haskellLib.cleanGit: /nix/store/3357wa1ysx8h8m3m4kfcy4a9pwv7msiw-ormolu-src does not seem to be a git repository,
assuming it is a clean checkout.
trace: No index state specified for haskell-project, using the latest index state that we know about (2022-02-15T00:00:00Z)!
trace: No index state specified for cabal-install, using the latest index state that we know about (2022-09-07T00:00:00Z)!
trace: No index state specified for ghcid, using the latest index state that we know about (2022-09-07T00:00:00Z)!
trace: No index state specified for haskell-language-server, using the latest index state that we know about (2022-09-07T00:00:00Z)!
/nix/store/kcf8p80lyri0w8k0viwcik3wz238q57s-stdenv-darwin/setup: line 102: warning: setlocale: LC_ALL: cannot change locale (C.UTF-8): No such file or directory
make: *** No targets specified and no makefile found.  Stop.
````

Related to _Originally posted by @JaredCorduan in https://github.com/input-output-hk/cardano-ledger/issues/3074#issuecomment-1276059386_

--- a/README.md
+++ b/README.md
@@ -124,12 +124,14 @@ for the ledger specification corresponding to the Shelley release, or
 the Byron release). Then, build the latex document by running:
 
 ```shell
+cd <myLaTexDir>
 nix-shell --pure --run make
 ```
 
 For a continuous compilation of the `LaTeX` file run:
 
 ```shell
+cd <myLaTexDir>
 nix-shell --pure --run "make watch"
 ```
 

